atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -736,6 +738,80 @@ public void testDeleteMultipleVersions() {
     }
 
     @Test
+    public void testDeleteRangeReverse() {
+        if (reverseRangesSupported()) {
+            putTestDataForMultipleRows(); // set up rows row0,row1,row2
+            RangeRequest range = RangeRequest.builder(true)
+                    .startRowInclusive("row1b".getBytes())
+                    .endRowExclusive(row0)
+                    .build();
+            // should delete only row1
+            keyValueService.deleteRange(TEST_TABLE, range);
+
+            List<byte[]> keys = Lists.newArrayList();
```

Comment:
The four-line check is duplicated a few times. You can factor it out and just pass in a list (or varargs parameter) of which values should remain.